### ISSUE TITLE
Documentation: matplotlibrc example file confuses x and yticks.

### ISSUE THREAD

**Issue by ImportanceOfBeingErnest** [matplotlib/matplotlib#8428]
There is a small mistake in the current version of the Customizing matplotlib ([http://matplotlib.org/users/customizing.html](http://matplotlib.org/users/customizing.html)) article.

In the current version the example matplolitbrc file  says in the section where **y**ticks are set

```
#xtick.major.left     : True   # draw y axis left major ticks
#xtick.major.right    : True   # draw y axis right major ticks
#xtick.minor.left     : True   # draw y axis left minor ticks
#xtick.minor.right    : True   # draw y axis right minor ticks

```
while it should of course be **y**tick.m......

While this is a pretty obvious thing, someone might still want to change it at some point.
(I was unable to locate the file in the code on GitHub, so I can't make a pull request myself.)

**Comment by NelleV** [comment]
Hi @ImportanceOfBeingErnest 
Thanks for reporting this issue. Do you mind opening a PR fixing this?
The file is located in the root folder and is called `matplotlibrc.template`

**Comment by WeatherGod** [comment]
Double-check that it hasn't been fixed already. I could have sworn this was
spotted and fixed recently.

**Comment by ImportanceOfBeingErnest** [comment]
Yep, it has been fixed by #8042 
So it should be online quite soon I guess?!

It was just confusing because the link in the customizing `customizing.rst` file points to a file which doesn't seem to exist ( `(download) <../_static/matplotlibrc>__`).

**Comment by NelleV** [comment]
Great! Thanks @ImportanceOfBeingErnest 

Indeed, it is sometimes hard to find files in Matplotlib's source tree, as it is very unstandard and complicated. This is why we are here to help :)
